packages/oas-to-snippet/package.json

```diff
@@ -61,7 +61,6 @@
     "httpsnippet-client-api": "^7.0.0-beta.4",
     "oas": "file:../oas",
     "tsup": "^8.0.2",
-    "type-fest": "^4.18.3",
```

Comment:
wonder if we should add `knip` into this repo

Reply:
yeah, will in a followup